Training.DAL/Context/TrainingDbContext.cs

```diff
@@ -39,6 +39,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 x.HasKey(x => x.Id);
                 x.HasIndex(x => x.ISBN).IsUnique();
                 x.Property(x => x.ISBN).HasMaxLength(15);
+                x.HasOne(x => x.Album).WithMany(x => x.Books);
```

Comment:
Perhaps we are having a different concept of "Album", I see like a music album, and being something completly isolated, I think the relation with Book is not necessary

However, despite that for me functionally it has no sense, technically speaking, the relation with EFCore has been correctly done ;)